.github/workflows/gradle.yml

```diff
@@ -21,6 +21,6 @@ jobs:
         cache: 'gradle'
         check-latest: true
     - name: Validate Gradle wrapper
-      uses: gradle/wrapper-validation-action@v2
+      uses: gradle/actions/wrapper-validation@v3
```

Comment:
Good catch, do you want to do this in a separate PR?

Reply:
See #15